token/services/db/sql/common/tokenlock.go

```diff
@@ -40,26 +40,18 @@ func newTokenLockDB(readDB, writeDB *sql.DB, tables tokenLockTables) *TokenLockD
 	}
 }
 
-func NewTokenLockDB(readDB, writeDB *sql.DB, opts NewDBOpts) (*TokenLockDB, error) {
-	tables, err := GetTableNames(opts.TablePrefix)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get table names")
-	}
-
-	tokenLockDB := newTokenLockDB(
+func NewTokenLockDB(readDB, writeDB *sql.DB, tables tableNames) (*TokenLockDB, error) {
+	return newTokenLockDB(
 		readDB,
 		writeDB,
 		tokenLockTables{
 			TokenLocks: tables.TokenLocks,
 			Requests:   tables.Requests,
-		},
-	)
-	if opts.CreateSchema {
-		if err = common.InitSchema(writeDB, []string{tokenLockDB.GetSchema()}...); err != nil {
-			return nil, err
-		}
-	}
-	return tokenLockDB, nil
+		}), nil
+}
+
+func (db *TokenLockDB) CreateSchema() error {
```

Comment:
extracted CreateSchema in order to be aligned with the FSC db's and re-use the configs.